src/classes/ALLO_Multicurrency_TDTM.cls

```diff
@@ -194,12 +172,9 @@ public class ALLO_Multicurrency_TDTM extends TDTM_Runnable {
     * @param amt A decimal of the amount to convert.
     * @return decimal The converted amount.
     ********************************************************************************************************/
-        //do a conversion from amount in corporate currency to the currency specified by the ISO code
+    //do a conversion from amount in corporate currency to the currency specified by the ISO code
     public static decimal convertFromCorporate(String ISO,Decimal amt) {
```

Comment:
Where else are these referenced? If not many, is the method still necessary or can we just call UTIL_ConvertCurrency. directly?